drb/dockerscripts/functions.sh

```diff
@@ -42,11 +42,11 @@ function setup_rpm_builddeps {
 function map_uid_gid_to_existing_users {
     #rpmbuild complains if it can't find a proper user for uid/gid of the source files;
     #we should add all uid/gids for source files.
-    for gid in $(find ${TOMAP_DIR} | xargs stat -c '%g' | sort | uniq); do
+    for gid in $(find ${TOMAP_DIR} | while read -r filename; do stat -c '%g' "$filename"; done | sort | uniq); do
```

Comment:
hi @rassie, 
With additional options, commands `find` and `xargs`  work well with spaces: 
it's better than using a while loop :
`find ${TOMAP_DIR} -print0 | xargs -0 stat -c '%g' ...`

(same thing about line  with `stat -c '%u'`)